packages/templates/nextjs-marketing-demo/src/components/CustomRatingComponentRegistration.tsx

```diff
@@ -0,0 +1,52 @@
+import { Rate } from 'antd';
+type CustomRatingComponentProps = React.ComponentProps<typeof Rate>;
+
+export const CustomRatingComponentRegistration = {
+  component: (props: CustomRatingComponentProps) => {
+    const { disabled, value, ...rest } = props;
+    return <Rate disabled={disabled} value={value} {...rest} />;
+  },
+  definition: {
+    id: 'custom-rating',
+    name: 'Custom Rating',
+    category: 'Custom Components',
+    variables: {
+      disabled: {
+        displayName: 'Read only',
+        type: 'Boolean',
+        defaultValue: false,
```

Comment:
I think we might want to make the Read only true by default since I don't really see anyone interacting with for the demo